src/main.d.ts

```diff
@@ -2,7 +2,7 @@
  * @deprecated Use AciContractCallEncoder
  */
 export class Encoder {
-  constructor(aci: { [key: string]: any });
+  constructor(aci: any[]);
```

Comment:
I've added this in #158 probably because I get confused by aci in terms of http compiler